client/app/bundles/course/achievement/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import { reducer as formReducer } from 'redux-form';
+import editPage from './editPage';
+import formDialog from './formDialog';
+
+export default combineReducers({
+  formDialog,
```

Comment:
How about `indexFormDialog`? It isn't the entire `indexPage` yet (unlike `editPage`), which is why I'm a little hesitant to name it `indexPage`.